jbmc/regression/jbmc-strings/java_append_char/test_append_char.java

```diff
@@ -1,20 +1,34 @@
+// This test uses CProverString so should be compiled with
+// javac test_append_char.java ../cprover/CProverString.java ../cprover/CProver.java
+
 public class test_append_char
 {
-  public static void main(boolean b)
-   {
-      char[] diff = {'d', 'i', 'f', 'f'};
-      char[] blue = {'b', 'l', 'u', 'e'};
-
-      StringBuilder buffer = new StringBuilder();
+    public static String ofCharArray(char value[], int offset, int count) {
```

Comment:
I'm not sure what this whole change is about. The regression is not clear as to its purpose. Also you mention the test uses `CProverString` but I don't see where.

Reply:
Also I fail to see how the commit and the commit message are related.

Reply:
Yes you are right, I updated java-models-library so that we don't have to change this test.